lib/travis/hub/api/sentry.rb

```diff
@@ -0,0 +1,12 @@
+require 'raven'
+
+module Travis
+  module Hub
+    class Sentry < Sinatra::Base
+      configure do
+        Raven.configure { |c| c.tags = { environment: environment } }
```

Comment:
Would be really nice to also have `x_request_id: env['HTTP_X_REQUEST_ID'])`, the way we do in API.